base/docs/helpdb/Dates.jl

```diff
@@ -49,11 +50,55 @@ All characters not listed above are treated as delimiters between date and time
 Dates.DateTime(dt::AbstractString, format::AbstractString)
 
 """
+    format(dt::TimeType, format::AbstractString; locale="english") -> AbstractString
+
+Construct a string by using a `TimeType` and applying the pattern provided by the
```

Comment:
Of course not. If you start jumping to the next issue that way, you will end up fixing bugs in LLVM. :-)